src/shapleypy/coalition.py

```diff
@@ -15,8 +15,11 @@
 
 
 class Coalition:
-    def __init__(self, id: int) -> None:
-        self.id = np.uint32(id)
+    def __init__(self, id: int | np.uint32) -> None:
```

Comment:
This does not solve #15. This is just something for "comfort" of development. Since internally I decided to use numpy.uint32 for ID of Coalition it makes sense to enable it for initialization (when developing back end I sometimes have variable that is numpy.uint32 so it does not make sense to retype to int and immediately back to numpy.uint32). But also I do not want force users to initialize it with numpy.uint32, because it makes things harder (for example Coalition(0b0010) would have to look like Coalition(numpy.uint32(0b0010)))